info.femm.unofficial.appdata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>info.femm.unofficial</id>
+  <name>FEMM 4.2</name>
+  <summary>Finite Element Method Magnetics</summary>
+  <metadata_license>Aladdin Free Public License</metadata_license>
```

Comment:
this is an invalid license for the **metadata**. `CC0-1.0` is prefered.

This file does NOT validate, it's missing too many fields, including the project license, content rating, screenshots. release.

Reply:
also I'm not sure "unofficial" is adequate for the application name part of the ID.

Reply:
Ok. I really have trouble to find an example of how to build the xml. Do you have any template? I will try to fix everything.
About the name, do you have a suggestion as good practice?

Reply:
https://github.com/flathub/flathub/wiki/App-Requirements#appstream